.drone.yml

```diff
@@ -0,0 +1,28 @@
+kind: pipeline
+type: kubernetes
+name: default
+
+steps:
+  - name: test-opa
+    image: alpine
+    commands:
+      - apk add jq
+      - latest=$(wget -q -O- "https://api.github.com/repos/open-policy-agent/opa/releases/latest" | jq -r .tag_name)
+      - wget -O opa https://github.com/open-policy-agent/opa/releases/download/$latest/opa_linux_amd64
+      - chmod +x opa
+      - ./opa test --ignore *.yaml --ignore *.json -v base
```

Comment:
Could you use `image: openpolicyagent/opa:latest` to achieve the same thing?

Reply:
I've tried but drone runs the commands in sh, and the opa image does not have it (also tried with the debug image)

Reply:
This works for me locally: 
```
  - name: test-opa
    image: openpolicyagent/opa:latest
    command:
      - test 
      - --ignore=*.yaml
      - --ignore=*.json
      - -v
      - base
```

Reply:
Well, works like a charm, I'll update